### PR TITLE
[4464] Update overview docs

### PR DESCRIPTION
**Pull request by bychkovdmitry**
### 🎯 Goal

Closes https://github.com/GetStream/stream-chat-android/issues/4464

Update overview docs.

### 🎨 UI Changes

screencapture-localhost-3000-chat-docs-sdk-android-compose-overview-2022-11-23-02_03_56

### 🧪 Testing

Launch docusaurus with `npx stream-chat-docusaurus -i -s`, and ensure that the changes look good.

#### General
- [x] I have signed the [Stream CLA](https://docs.google.com/forms/d/e/1FAIpQLScFKsKkAJI7mhCr7K9rEIOpqIDThrWxuvxnwUq2XkHyG154vQ/viewform) (required)
- [x] Assigned a person / code owner group (required)
- [x] Thread with the PR link started in a respective Slack channel (#android-chat-core or #android-chat-ui) (required)
- [x] PR targets the `develop` branch
- [x] PR is linked to the GitHub issue it resolves

#### Code & documentation
- [ ] Changelog is updated with client-facing changes
- [ ] New code is covered by unit tests
- [ ] Comparison screenshots added for visual changes
- [x] Affected documentation updated (KDocs, docusaurus, tutorial)

### ☑️Reviewer Checklist
- [ ] UI Components sample runs & works
- [ ] Compose sample runs & works
- [ ] UI Changes correct (before & after images)
- [ ] Bugs validated (bugfixes)
- [ ] New feature tested and works
- [ ] Release notes and docs clearly describe changes
- [ ] All code we touched has new or updated KDocs

### CODE REVIEW

File: docusaurus/docs/Android/03-ui/01-overview.mdx [hunk 1]
```diff
@@ -39,11 +48,11 @@ Since this library uses Material elements, make sure that you use a Material the
 <style name="AppTheme" parent="Theme.MaterialComponents.DayNight.NoActionBar">
 ```
 
-If you want to keep using an `AppCompat` theme for styling, you can use a [Bridge Theme](https://github.com/material-components/material-components-android/blob/master/docs/getting-started.md#bridge-themes) to support using Material based components at the same time.
+If you want to keep using an `AppCompat` theme for styling, you can inherit from a [Bridge Theme](https://m2.material.io/develop/android/docs/getting-started#bridge-themes) to support using Material based components.
```
Comment (bychkovdmitry) [inline docusaurus/docs/Android/03-ui/01-overview.mdx:51]: They don't mention bridge theme anymore anywhere in https://github.com/material-components/material-components-android

**Review by skydoves — APPROVED** [review]
Looks good to me!

File: stream-chat-android-docs/src/main/java/io/getstream/chat/docs/java/ui/Overview.java [hunk 2]
```diff
@@ -1,27 +1,33 @@
...
+    private MessageListView messageListView;
+
+    /**
+     * [ViewModels](https://getstream.io/chat/docs/sdk/android/ui/overview/#viewmodels)
+     */
+    public void viewModels() {
+        // 1
+        ViewModelProvider.Factory factory = new MessageListViewModelFactory
                 .Builder()
-                .cid("channelType:channelId")
```
Comment (MarinTolic) [inline stream-chat-android-docs/src/main/java/io/getstream/chat/docs/java/ui/Overview.java:20]: I think I actually prefer `channelType:channelId` as it is more explicit.
What do you think?

**Review by MarinTolic — APPROVED** [review]

File: stream-chat-android-docs/src/main/java/io/getstream/chat/docs/java/ui/Overview.java (hunk 2 above)
Comment (bychkovdmitry) [reply stream-chat-android-docs/src/main/java/io/getstream/chat/docs/java/ui/Overview.java:20]: I'm leaving `messaging:123` as it's slightly more common in our codebase.

**Pull request merged**